config/_shared.json

```diff
@@ -132,5 +132,6 @@
 	"enable_all_sections": true,
 	"sections": {
 		"jetpack-cloud": false
-	}
+	},
+	"theme": "default"
```

Comment:
Anyway, what does the `default` theme do? It doesn't seem like it's doing anything?

Reply:
yeah, right now it doesn't do anything.